### PR TITLE
[TDV-535] Handle a default limit max value

### PR DESCRIPTION
**Pull request by EddyRock**
https://tokend.atlassian.net/browse/TDV-535?filter=-1

### CODE REVIEW

File: src/components/User/Limits/Limits.Reviewer.vue [hunk 1]
```diff
@@ -302,17 +310,29 @@ export default {
       }
     },
     newLimit () {
+      const requestDetails = this.desiredLimitDetails
+      const operationType = OPERATION_TYPES[requestDetails.operationType]
+      const limitsKeys = Object.keys(this.desiredLimitDetails.limits)
+      const limits = {}
+      limitsKeys.forEach(limit => {
```
Comment (EddyRock) [inline src/components/User/Limits/Limits.Reviewer.vue]: for of object.entries()

File: src/components/User/Limits/Limits.Reviewer.vue [hunk 2]
```diff
@@ -302,17 +310,29 @@ export default {
       }
     },
     newLimit () {
+      const requestDetails = this.desiredLimitDetails
+      const operationType = OPERATION_TYPES[requestDetails.operationType]
```
Comment (EddyRock) [inline src/components/User/Limits/Limits.Reviewer.vue]: 2 unused

File: src/components/User/Limits/Limits.Reviewer.vue [hunk 3]
```diff
@@ -302,17 +310,29 @@ export default {
...
         assetCode: this.assetCode,
-        statsOpType: STATS_OPERATION_TYPES[OPERATION_TYPES[this.desiredLimitDetails.operationType]]
+        statsOpType: STATS_OPERATION_TYPES[operationType]
       }
     },
     uploadedDocuments () {
       if (!this.desiredLimitDetails.documents) return
       return this.desiredLimitDetails.documents
+    },
+    getRequestType () {
```
Comment (EddyRock) [inline src/components/User/Limits/Limits.Reviewer.vue]: remove "get"

File: src/components/User/Limits/Limits.Reviewer.vue [hunk 4]
```diff
@@ -321,7 +341,8 @@ export default {
     async getRequest () {
       this.$store.commit('OPEN_LOADER')
       const request = await api.requests.get(this.id)
-      const requestDetails = request.details[snakeToCamelCase(request.details.requestType)].details
+      const camelCaseByType = snakeToCamelCase(request.details.requestType)
```
Comment (EddyRock) [inline src/components/User/Limits/Limits.Reviewer.vue]: camelCasedRequestType

File: CHANGELOG.md [hunk 5]
```diff
@@ -10,6 +10,7 @@ for further information about branching and tagging conventions.
 ## [Unreleased]
 #### Fixed
 - Fixed operation empty and loading list displaying of a user
+- Add validation in reviewLimitsUpdateRequest when parsing limits values
```
Comment (EddyRock) [inline CHANGELOG.md]: you do not do any validation actually

File: src/components/User/Limits/Limits.Reviewer.vue [hunk 6]
```diff
@@ -319,33 +316,36 @@ export default {
...
-      this.desiredLimitDetails = limitRequest
-        .details[requestType].details || '{}'
-    } catch (e) {
-      ErrorHandler.processWithoutFeedback(e)
-    }
+    await this.getRequest()
+    const limitRequest = await api.requests.get(this.id)
+    this.desiredLimitDetails = limitRequest
+      .details[snakeToCamelCase(limitRequest.details.requestType)].details ||
+      '{}'
```
Comment (max-nes) [inline src/components/User/Limits/Limits.Reviewer.vue]: restore try-catch

File: CHANGELOG.md [hunk 7]
```diff
@@ -10,6 +10,8 @@ for further information about branching and tagging conventions.
 ## [Unreleased]
 #### Fixed
 - Fixed operation empty and loading list displaying of a user
+- Add filter for fill limits in reviewLimitsUpdateRequest when 
+parsing limits values
```
Comment (max-nes) [inline CHANGELOG.md]: cannot get the meaning out of this. can you pls rephrase it? what actually was fixed?

**Pull request merged**